check-vmware/test_checks.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+import random
+import yaml
+
+from pyVmomi import vim
+from vmware_checks import CHECKS
+from vmware_logconf import get_logger
+from wrapanapi.systems.virtualcenter import VMWareSystem
+
+
+@pytest.fixture(scope="session")
+def provider_data():
+    try:
+        with open("conf/cfme-qe-yamls/complete/cfme_data.yaml", "r") as stream:
+            cfme_data = yaml.safe_load(stream)
+    except IOError:
+        pytest.fail("CFME Data YAML file not found, no provider data available.")
+    else:
+        providers = cfme_data.get("management_systems")
+        vsphere_providers = {
+            key: providers[key] for key in list(providers.keys())
+            if providers[key].get("type") == "virtualcenter"
+        }
+        yield vsphere_providers.get(random.choice(list(vsphere_providers.keys())))
+
+@pytest.fixture(scope="session")
+def credentials(provider_data):
+    try:
+        with open("conf/cfme-qe-yamls/complete/credentials.yaml", "r") as stream:
```

Comment:
Where are we handling decryption ?

Reply:
I didn't include decryption because I didn't want to reproduce too much code from `integration-tests` do you think we should include decryption?

Reply:
So you are proposing to decrypt the creds manually before running tests?

Reply:
Thanks for update.